app/globals.css

```diff
@@ -34,3 +38,10 @@
 .cardBoxShadowHover {
   @apply transition-transform duration-300 ease-in-out hover:scale-105 hover:shadow-2xl dark:bg-black-200 dark:shadow-cardShadowSmall dark:hover:shadow-cardShadowLarge;
 }
+
+.button-hover:hover .translate-image {
```

Comment:
If you need help applying this within your tailwind, try the following... This should work, but you will have to test it. 

```
<div class="button-hover group">
  <div class="translate-image group-hover:translate-x-2">
    <!-- Image or content goes here -->
  </div>
</div>
```

By using the `group-hover:translate-x-2` utility class in conjunction with the group class on the parent, you're leveraging Tailwind's native capabilities to achieve the desired hover effect.